runtime/ftplugin/man.vim

```diff
@@ -1,37 +1,26 @@
-" Vim filetype plugin file
-" Language:	man
-" Maintainer:	SungHyun Nam <goweol@gmail.com>
-
-if has('vim_starting') && &filetype !=# 'man'
-  finish
-endif
-
-" Only do this when not done yet for this buffer
 if exists('b:did_ftplugin')
   finish
 endif
 let b:did_ftplugin = 1
 
-" Ensure Vim is not recursively invoked (man-db does this)
-" when doing ctrl-[ on a man page reference.
-if exists('$MANPAGER')
-  let $MANPAGER = ''
-endif
```

Comment:
Yes, the problem is known, but we can't just remove it. It was discussed here:

https://gitter.im/neovim/neovim?at=5645154210f6aea94fdd7f08